packages/thumbprint-react/components/Image/index.tsx

```diff
@@ -22,7 +22,8 @@ type ImageSource = {
     srcSet: string;
 };
 
-export interface ImageProps {
+export interface ImageProps
+    extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement> {
```

Comment:
Should this be HTMLPictureElement, instead of the generic HTMLElement?

Reply:
I don't think so. When I hover on `<picture`, I get this type definition:
![image](https://github.com/thumbtack/thumbprint/assets/916038/5c9b3215-04c5-48fd-b914-99a516b304d7)

Reply:
Ah interesting, yeah I think that's technically a bug in the DOM typings, though in practice it doesn't matter because <picture> doesn't have any custom attributes, so it's equivalent to the generic interface. We can stick with this for consistency with the DOM typings